### ISSUE TITLE
Mark start/finish line on tracks that don't have a marking

### ISSUE THREAD

**Issue by Wuzzy2** [supertuxkart/stk-code#2135]
A few tracks don't have any markings for the exact position of the start/finish. It would be useful to have some sort of marking (either a gate or a line on the ground or both). Therefore, I kindly request this here.

The tracks are:
- [x] Around the Lighthouse
- [x] that ghost mansion
- [x] Fort Magma
- [x] maybe: (Cocoa Temple (actually it has a large gate, but it is hard to see/estimate the actual line because the road is very broad. I drawn line might help).)
- [x] Gran Paradiso Island (see Cocoa Temple)

See also: #2134

**Comment by auriamg** [comment]
I don't really see the issue in cocoa temple, the gate is pretty clear. Why would you need to know the precise line location?

Agreed about lighthouse / mansion / fort magma, though.  As we remake these tracks, the start line should be made clearer

**Comment by hiker** [comment]
I agree with this ticket. First of all, it is in our style guide (http://supertuxkart.sourceforge.net/Style_Guidelines#Track_guidelines_for_the_official_packages):
"Make sure you add a starting line texture (checkered line) so that people know where the finish line is. Preferably use the check.png file as a texture, but if it doesn't go well with the style of the track, feel free to use a different style. "
I've requested those lines over and over, but they are never added.

The lines are important to easily show when a race is finished. The 'gates' in various tracks are not visible that close to the finish line anymore (plus they are just too far away to be useful). Additionally, once we merge in the replay gsoc project, the idea was also to have a photo-shoot finish/replay - which needs a clearly visible line on the ground. We have already modified the code so that the finish detection is based on the front of the kart, and not on the middle (which could lead to incorrect results since karts have different lengths).
